code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -471,3 +471,5 @@ emp_act
 	..()
 	if(temperature >= 330)	bodytemperature = bodytemperature + (temperature - bodytemperature)
 	if(temperature <= 280)	bodytemperature = bodytemperature - (bodytemperature - temperature)

```

Comment:
Why not move the temperature changes for humans to the base level of species?

The more abstract the `human` mob is and the more "concrete" `species` datums are, the better.